### PR TITLE
fix(deploy_script): to set REALM env variable

### PR DESCRIPTION
**Pull request by corinnekrych**

### CODE REVIEW

File: config/webpack.dev.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const ENV = process.env.ENV || process.env.NODE_ENV || 'development';
 const API_URL = process.env.API_URL || (ENV === 'inmemory' ? 'app/' : 'http://localhost:8080/api/');
 const FORGE_URL = process.env.FORGE_URL || 'http://localhost:8080/forge';
 const FABRIC8_WIT_API_URL = process.env.FABRIC8_WIT_API_URL || 'http://localhost:8080/api/';
-const REALM = process.env.REALM || "fabric8";
+const FABRIC8_REALM = process.env.FABRIC8_REALM || "fabric8";
```
Comment (hectorj2f) [inline config/webpack.dev.js]: @corinnekrych could you add KEYCLOAK keyword to the env variable name ? I think people will quickly identify to which component belongs.

**Comment by hectorj2f** [comment]
+1 on this change

**Comment by alexeykazakov** [comment]
> Could you name this variable FABRIC*_KEYCLOAK_REALM ? Similarly how it is done for the core configuration such as FABRIC8_WIT_API_URL.

I would avoid using KEYCLOAK names because it's implementation specific. Use SSO instead.

**Comment by hectorj2f** [comment]
@alexeykazakov +1 on using the SSO

**Comment by corinnekrych** [comment]
@alexeykazakov @hectorj2f After discussing with @xcoulon we agreed upon FABRIC8_REALM I've renamed it already in the 2 other repos: fabric8-ui, fabric8-recommender.

**Comment by hectorj2f** [comment]
I don't find this ENV variable descriptive enough but I don't want it to be a blocker

**Comment by alexeykazakov** [comment]
Yeah.. FABRIC8_SSO_REALM or just SSO_REALM would be better but I don't to block it either. FABRIC8_REALM is good enough.

**Comment by nimishamukherjee** [comment]
@corinnekrych please rebase this PR. These files have been moved under the runtime folder.

**Comment by corinnekrych** [comment]
[test]

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1913-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1913-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1913-fabric8-planner.int.rdu2c.fabric8.io

**Comment by corinnekrych** [comment]
@ldimaggi any idea why the integration test are failing? This PR is to introduced a new environment variable which has a default value in  runtime/config/webpack.test.js.

**Comment by nimishamukherjee** [comment]
[test]

**Comment by fabric8cd** [comment]
@corinnekrych snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1913-fabric8-planner.int.rdu2c.fabric8.io

**Pull request merged**